bin/update_tarball-install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+DEST=/usr/local/repo/tarball-install
+
+#############################
+### Mirror tarball-client ###
+#############################
+
+/usr/bin/lftp -c mirror                                              \
+  --delete                                                           \
+  --exclude-glob=osg-afs-client-*         `# Skip osg-afs-client`    \
+  --exclude-glob=*tarballs.rescue         `# Ignore rescue tarballs` \
+  --exclude-glob=osg-wn-client-latest.*   `# Ignore latest symlinks` \
+  --exclude=3.2                           `# Exclude old releases`   \
+  https://vdt.cs.wisc.edu/tarball-client/                            \
+  $DEST
+
+##############################
+### Create latest symlinks ###
+##############################
+
+# For each OSG release series directory
+for REL_DIR in "$DEST"/* ; do
+
+  # For each "release/architecture"
+  #   E.g. /usr/local/repo/tarball-install/3.3/i386/
+  for REL_ARCH_DIR in "$REL_DIR"/* ; do
+
+    # For tarballs inside...
+    #   E.g. /usr/local/repo/tarball-install/3.3/i386/osg-wn-client-3.3.5-1.el6.i386.tar.gz
+    # Cut the last 4 fields and make unique os.arch list
+    OS_ARCHES=$(find "$REL_ARCH_DIR" -type f | rev | cut -d. -f1-4 | rev | sort | uniq)
+
+      # For each os.arch suffix
+      #   E.g. el7.x86_64.tar.gz
+      for OS_ARCH in $OS_ARCHES ; do
+
+        # Find the latest tarfile
+        LATEST=$(find "$REL_ARCH_DIR" -name "*.$OS_ARCH" | sort --version-sort | tail -1)
+
+        # And create latest symlink
+        ln -s --relative --force $LATEST $REL_DIR/osg-wn-client-latest.$OS_ARCH
```

Comment:
The slightly nervous side of me would quote these variables, which are based on the contents of directories:

```suggestion
        ln -s --relative --force "$LATEST" "$REL_DIR/osg-wn-client-latest.$OS_ARCH"
```

(Maybe not an issue if root owns the directories, but in general someone creating an extra file or two in these directories can cause a lot of trouble if you don't bother to quote the variable where they are used.)

Reply:
Much appreciated. Committed along with the spacing changes.